pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -90,29 +100,13 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 		klog.V(4).Infof("Added Job <%s/%s> into Queue <%s>", job.Namespace, job.Name, job.Queue)
 		jobsMap[job.Queue].Push(job)
 	}
+}
 
-	klog.V(3).Infof("Try to allocate resource to %d Queues", len(jobsMap))
-
+func (alloc *Action) allocateResources(queues *util.PriorityQueue, jobsMap map[api.QueueID]*util.PriorityQueue) {
```

Comment:
This method does two things, firstly pickUpTasks and allocate task secondly, so rename this method name to pickUpTasks  and rename processTasks to allocateResourcesForTasks, and both make them as a dependent method seperately is better, just be consistent with `alloc.pickUpQueuesAndJobs`.

Reply:
The main logic of the `allocateResources` method is in a **for loop**, and a longer part of the logic within the loop has been separated into `processTasks`. I attempted to decouple these two methods and have them both called by the `Execute` method, but this failed.

Moving the for loop logic into `Execute` essentially means removing the `allocateResources` method, which would significantly complicate `Execute`. Therefore, I think it's not suitable to rename `allocateResources` to `pickUpTasks`.

As for renaming `processTasks`, I am open to it. If you think a change would be better, I can add another commit to complete this renaming.

Reply:
Got that.